### PR TITLE
Travel advice callout changes - July 15 2021

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/P881xXrT

# Before

Screenshot 2021-07-16 at 11 01 40

# After

Screenshot 2021-07-16 at 11 22 01

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/views/travel_advice/index.html.erb [hunk 1]
```diff
@@ -27,16 +27,14 @@
             </div>
             <div class="travel-advice-notice__content">
               <p class="govuk-body">
-                Follow current COVID-19 rules where you live: <a href="/guidance/covid-19-coronavirus-restrictions-what-you-can-and-cannot-do" class="govuk-link">England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/" class="govuk-link">Scotland</a>, <a href="https://gov.wales/coronavirus-travel" class="govuk-link">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you" class="govuk-link">Northern Ireland</a>.
+                To understand the risks in a country, including the latest COVID restrictions (including for entry), follow FCDO Travel Advice.
```
Comment (kevindew) [inline app/views/travel_advice/index.html.erb:30]: I notice on the design this has a link to this. It seems the right decision to not link here but I wonder if this is going to surprise the people who put the copy together as they might not have realised this quirk.

File: app/views/travel_advice/index.html.erb [hunk 2]
```diff
@@ -27,16 +27,14 @@
...
               </p>
               <p class="govuk-body">
-                To understand the risks in a country follow <a href="/guidance/travel-advice-novel-coronavirus" class="govuk-link">FCDO Travel Advice</a>.
-              </p>
-              <p class="govuk-body">
-                When you return, follow the rules to <a href="/uk-border-control" class="govuk-link">enter the UK from abroad</a> (except from Ireland).
+                Check what you need to do to <a href="//guidance/travel-abroad-from-england-during-coronavirus-covid-19" class="govuk-link">travel abroad and return to England</a>, or read travel guidance for <a href="https://www.gov.scot/publications/coronavirus-covid-19-guidance-on-travel-and-transport/" class="govuk-link">Scotland</a>, <a href="https://gov.wales/coronavirus-travel" class="govuk-link">Wales</a> or <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-travel-advice" class="govuk-link">Northern Ireland</a>.
```
Comment (kevindew) [inline app/views/travel_advice/index.html.erb:37]: You've got a double slash in the first link on this line.

**Review by kevindew — COMMENTED** [review]
Thanks, looks good. I've caught one minor issue and got a question for another.

File: app/views/travel_advice/index.html.erb (hunk 1 above)
Comment (leenagupte) [reply app/views/travel_advice/index.html.erb:30]: We've spotted this before, and always remove the link on the search page as it links to itself and is a worse user experience.

Comment (kevindew) [reply app/views/travel_advice/index.html.erb:30]: Great - glad there's precedence.

File: app/views/travel_advice/index.html.erb (hunk 2 above)
Comment (leenagupte) [reply app/views/travel_advice/index.html.erb]: Good spot! Updated and fixed up.

**Review by kevindew — APPROVED** [review]

**Pull request merged**